server/tests/geocodes.test.js

```diff
@@ -0,0 +1,76 @@
+/* eslint-disable no-restricted-syntax, no-await-in-loop */
+const { v4 } = require('uuid');
```

Comment:
Seems like there are some unused variables in this doc due to the commented block